runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpPanel.java

```diff
@@ -112,6 +114,24 @@
 		popupMenu.add(resetPerHour);
 		popupMenu.add(pauseAll);
 		popupMenu.add(unpauseAll);
+		popupMenu.addPopupMenuListener(new PopupMenuListener()
+		{
+			@Override
+			public void popupMenuWillBecomeVisible(PopupMenuEvent popupMenuEvent)
+			{
+				openXpTracker.setVisible(xpTrackerConfig.wiseOldManOpenOption());
+			}
+
+			@Override
+			public void popupMenuWillBecomeInvisible(PopupMenuEvent popupMenuEvent)
+			{
+			}
+
+			@Override
+			public void popupMenuCanceled(PopupMenuEvent popupMenuEvent)
+			{
+			}
```

Comment:
Why have you added overrides to these methods?

Reply:
If I don't, I get these errors:
- <anonymous net.runelite.client.plugins.xptracker.XpPanel$1> is not abstract and does not override abstract method popupMenuCanceled(javax.swing.event.PopupMenuEvent) in javax.swing.event.PopupMenuListener
- <anonymous net.runelite.client.plugins.xptracker.XpPanel$1> is not abstract and does not override abstract method popupMenuWillBecomeInvisible(javax.swing.event.PopupMenuEvent) in javax.swing.event.PopupMenuListener

XpInfoBox.java does the same thing.